llms/mlx_lm/generate.py

```diff
@@ -35,41 +34,15 @@ def setup_arg_parser():
         "--temp", type=float, default=DEFAULT_TEMP, help="Sampling temperature"
     )
     parser.add_argument("--seed", type=int, default=DEFAULT_SEED, help="PRNG seed")
+    parser.add_argument('-v', '--verbose', help='Increase output verbosity', action='store_true')
     return parser
 
 
 def main(args):
-    mx.random.seed(args.seed)
     model, tokenizer = load(args.model)
-    print("=" * 10)
-    print("Prompt:", args.prompt)
-    prompt = tokenizer.encode(args.prompt)
-    prompt = mx.array(prompt)
-    tic = time.time()
-    tokens = []
-    skip = 0
-    for token, n in zip(
-        generate_step(prompt, model, args.temp), range(args.max_tokens)
-    ):
-        if token == tokenizer.eos_token_id:
-            break
-        if n == 0:
-            prompt_time = time.time() - tic
-            tic = time.time()
-        tokens.append(token.item())
-        s = tokenizer.decode(tokens)
-        print(s[skip:], end="", flush=True)
-        skip = len(s)
-    print(tokenizer.decode(tokens)[skip:], flush=True)
-    gen_time = time.time() - tic
-    print("=" * 10)
-    if len(tokens) == 0:
-        print("No tokens generated for this prompt")
-        return
-    prompt_tps = prompt.size / prompt_time
-    gen_tps = (len(tokens) - 1) / gen_time
-    print(f"Prompt: {prompt_tps:.3f} tokens-per-sec")
-    print(f"Generation: {gen_tps:.3f} tokens-per-sec")
+    output = generate(model, tokenizer, args.prompt, args.temp, args.max_tokens, args.seed, args.verbose)
+    if not args.verbose:
+        print(output)
```

Comment:
Let's just fix verbose to `True` for `generate.py`. I don't think anyone will want to use it without verbosity (that's meant more for the API)